config/routes.rb

```diff
@@ -23,6 +23,7 @@
   delete "/logout", to: "sessions#destroy"
   get "/dashboard", to: "users#show"
   post "update_quantity", to: "cart_staches#update_quantity"
+  get '/about', to: "static#about"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.